### DISCUSSION TITLE
Welcome to placemark-oss Discussions!

### DISCUSSION THREAD

**Discussion by tmcw** [placemark/placemark#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by okjuan** [comment]
Hi Tom, I'm Juan, nice to meet you :wave:

I don't have any experience with mapping software, but I'm very interested in data-viz and interfaces for learning/discovering e.g. stuff by [The Pudding](https://pudding.cool/), like their piece [How Music Taste Evolved](https://pudding.cool/2017/03/music-history/). I'm currently working on a web app called [Muze Radio](https://okjuan.me/muze-radio) that lets you explore Spotify's recommendations with fine-grained controls, but won't be publicly available until Spotify grants me an extended quota.

Anyway, I wanted to ask -- what is the intended use cases for Placemark? I can see that it's for collaborating on maps, but I'm not super clear on the main user flows. (Pardon the naive question.) You mention that it is like Figma for maps: is the goal of Placemark to produce an exported file (e.g. image, HTML, etc.)? Or is it meant as a workspace for people to brainstorm and analyze geo data?

Thanks!

**Reply by tmcw** [reply]
Hi @okjuan!

Sorry for the delay! The main usecase of Placemark is to create, edit, and export map data. It's for custom datasets, where you're deleting or moving or adding detail to geospatial objects.

It does have a lot of other functionality - there's some visualization capabilities, you can publish maps, convert between filetypes, and export to SVG. Those all work pretty well and are useful within the context of custom data, but Placemark isn't primarily a visualization tool - that's better-suited for something like Unfolded, Deck.gl, or Maplibre/Mapbox GL, or for smaller datasets, D3. For _basemap_ design, you could use something like Mapbox Studio.

**Comment by gcoch** [comment]
I'm happy to contribute for hosting costs of Placemark Play.  Best tool for updating, editing and doing QA on maps I've ever used.
